packages/jest-runner/package.json

```diff
@@ -17,6 +17,7 @@
     "@jest/console": "^26.6.2",
     "@jest/environment": "^26.6.2",
     "@jest/test-result": "^26.6.2",
+    "@jest/transform": "^26.6.2",
```

Comment:
Add to references in tsconfig

Reply:
It's present in `tsconfig.json`.
Sorry this line should have been part of #8751 , got missed in the rebase I'll be more careful next time 

https://github.com/facebook/jest/blob/68d68f5282bb11f93cc24ac41634667469138878/packages/jest-runner/tsconfig.json#L18